src/topology/uniform_space/compare_reals.lean

```diff
@@ -75,7 +75,8 @@ end
 
 /-- Cauchy reals packaged as a completion of ℚ using the absolute value route. -/
 noncomputable
-def rational_cau_seq_pkg : @abstract_completion ℚ $ is_absolute_value.uniform_space (abs : ℚ → ℚ) :=
+def rational_cau_seq_pkg : @abstract_completion ℚ $ is_absolute_value.uniform_space (abs :
+  ℚ → ℚ) :=
```

Comment:
What happened here? Don't split lines mid-argument. Better to split before the `(`.

Reply:
Must have accidentally hit Enter!